hr_attendance_schedule/models/hr_attendance_clock.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+import pytz
+from datetime import datetime
+
+from odoo import api, fields, models
+from odoo.tools.misc import DEFAULT_SERVER_DATETIME_FORMAT
+
+
+class AttendanceClock(models.AbstractModel):
+    _name = 'hr_attendance_schedule.clock'
+
+    @api.model
+    def get_system_clock(self):
+        now = datetime.strptime(fields.Datetime.now(), DEFAULT_SERVER_DATETIME_FORMAT)
+        return pytz.utc.localize(now).isoformat()
```

Comment:
this is the use case for https://github.com/odoo/odoo/blob/10.0/odoo/fields.py#L1514

note that `pytz.utc.localize(now)` is basically a no-op, and that your code will return tomorrow if you sit in Amsterdam and it's after 23.00. That's what context_timestamp is meant to fix

Reply:
Hmm, not exactly. The goal of the method is to return the current UTC date and time, but including the UTC time zone info at the end of the iso string. No user conversion needed.

`now.isoformat() = '2017-01-01T12:12:12'`
`pytz.utc.localize(now).isoformat() = '2017-01-01T12:12:12+00:00'`

I could also have just returned `now.isoformat() + '+00:00'` but coded this way feld more explicit. I see I may have been wrong.